src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,39 +1,67 @@
-import React from 'react';
+import React, { SetStateAction, useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  selectedTodo: Todo;
+  setSelectedTodo: React.Dispatch<SetStateAction<Todo | null>>;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  setSelectedTodo,
+}) => {
+  const [user, setUser] = useState<User>();
+  const [loadingUser, setLoadingUser] = useState<boolean>(false);
+
+  useEffect(() => {
+    setLoadingUser(true);
+
+    getUser(selectedTodo.userId)
+      .then(setUser)
+      .finally(() => setLoadingUser(false));
+  }, [selectedTodo.userId]);
+
   return (
     <div className="modal is-active" data-cy="modal">
```

Comment:
You are interacting with the DOM directly by using class names. It's better to avoid this and use React as much as possible. Consider using the `classnames` library for defining classes conditionally.